ui.apps/src/main/content/jcr_root/apps/custom_submit_action/invoke_service/post.POST.jsp

```diff
@@ -0,0 +1,27 @@
+<%@include file="/libs/fd/af/components/guidesglobal.jsp" %>
```

Comment:
@rakeshkumar-verma : whys is the file name having two post keywords ?

Reply:
@rishabh-behl  AF submit servlet only looks for post.POST.jsp script once form submited.